cognite_toolkit/_cdf_tk/prototypes/featureflag.py

```diff
@@ -1,20 +1,30 @@
-import os
+from __future__ import annotations
 
-from rich import print
+from pathlib import Path
 
+import yaml
 
-def enabled(flag: str) -> bool:
-    """
-    Check if a feature flag is enabled.
 
-    Args:
-        flag (str): The feature flag to check.
+class FeatureFlag:
+    _flags = None
 
-    Returns:
-        bool: True if the feature flag is enabled, False otherwise.
-    """
-    if os.environ.get(flag, "false").lower() == "true":
-        if enabled("FF_PRINT_FLAGS"):
-            print(f"[yellow]Feature flag {flag} is enabled.[/]")
-        return True
-    return False
+    @staticmethod
+    def _get_system_yaml_path() -> Path | None:
+        loc = Path(__file__)
+        while loc != loc.root:
+            loc = loc.parent
+            if (loc / "_system.yaml").exists():
+                return loc / "_system.yaml"
+        return None
```

Comment:
This will not work. The `featureflag.py` will be in the installation of `cognite-toolkit` that typically is far away from where the modules are located, for example, on my machine for the `maia` project `C:\Users\AndersAlbert\AppData\Local\pypoetry\Cache\virtualenvs\maia-N69zAN_F-py3.11\Lib\site-packages\cognite_toolkit\_cdf_tk\prototypes\featureflag.py`